modules/apps/login/login-web/src/main/resources/META-INF/resources/set_password.jsp

```diff
@@ -0,0 +1,200 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+
+//String currentURL = PortalUtil.getCurrentURL(request);
+jakarta.servlet.http.HttpServletRequest httpServletRequest = com.liferay.portal.kernel.util.PortalUtil.getOriginalServletRequest(com.liferay.portal.kernel.util.PortalUtil.getHttpServletRequest(renderRequest));
```

Comment:
Why do you need to use the class full name here instead of adding an import clause for the `httpServletRequest` variable?